app/views/browse/config-maps.html

```diff
@@ -10,7 +10,13 @@
             <div class="pull-right" ng-if="project && ('configmaps' | canI : 'create')">
               <a ng-href="project/{{project.metadata.name}}/create-config-map" class="btn btn-default">Create Config Map</a>
             </div>
-            <h1>Config Maps</h1>
+            <h1>Config Maps
+              <span class="page-header-link">
+                <a ng-href="{{'config-maps' | helpLink}}" target="_blank">
+                  Learn more <i class="fa fa-external-link" aria-hidden="true"></i>
```

Comment:
Probably should use a non-breaking space between the text and icon so that the icon doesn't ever wrap by itself. It could break on some longer headers at mobile.